### ISSUE TITLE
Support pkg:quiver v2

### ISSUE THREAD

**Issue by kevmoo** [dart-lang/angular_components#314]
Blocked on rolling it into Google3, obviously

**Comment by kevmoo** [comment]
ping @nshahan ?

**Comment by kevmoo** [comment]
Guessing we're still waiting on the roll internally – @grouma – know anything about this?

**Comment by nshahan** [comment]
Yeah needs an update internally. Is this really P1 High for this repo?

**Comment by kevmoo** [comment]
>  Is this really P1 High for this repo?

Eh...1.5. It's bad if our major frameworks don't support latest. Especially since quiver is "stable" as of v2

**Comment by rayk** [comment]
> +1
> Would be great as it is incompatible with the newest version of pageloader (3.0.2)

The current dependencies are certain contsrainting.

```bash
Resolving dependencies...
Because pageloader >=3.0.2 depends on quiver ^2.0.0 and angular_components ^0.9.0-beta+1 depends on quiver >=0.24.0 <0.30.0, pageloader >=3.0.2 is incompatible with angular_components ^0.9.0-beta+1.
So, because app depends on both angular_components ^0.9.1 and pageloader ^3.0.2, version solving failed.
```

**Comment by rayk** [comment]
Worth also noting that the current dependency on ``quiver >=0.24.0 <0.30.0 ``is blocking access to ``transformNullable`` in the ``quiver  2.0.1``. How really shure what the road block is here?

**Comment by nshahan** [comment]
This is unblocked now and should have a widened dependency in the next sync/version.

**Comment by nshahan** [comment]
Bam!
https://github.com/dart-lang/angular_components/blob/c6cb3d783e72ff1ed7974f18b7f169778c73d39b/angular_components/pubspec.yaml#L26

Published in v0.11.0
